src/GraphQL/Types/Schema.cs

```diff
@@ -25,7 +25,7 @@ public interface ISchema : IDisposable
 
     public class Schema : ISchema
     {
-        private readonly Lazy<GraphTypesLookup> _lookup;
+        protected readonly Lazy<GraphTypesLookup> _lookup;
```

Comment:
I will remove this change as I have removed the code my side that needed access to it.